tasks/copy_to_glacier/schemas/body.json

```diff
@@ -1,154 +1,156 @@
 {
-    "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Is this reformat intentional? We're getting a lot of diff noise.

Reply:
maybe it came from `black`